jax/experimental/jax2tf/tests/jax2tf_test.py

```diff
@@ -516,6 +516,15 @@ def jax_fn_array(x):
         tf_fn_array(np.array([3, 4, 5])), np.array([4.5, 10, 17.5],
                                                    jnp.bfloat16))
 
+  def test_kwargs(self):
+    # Re: https://github.com/google/jax/issues/6791
+    def f_jax(*, x):
+      return jnp.sum(x)
+    f_tf = jax2tf.convert(f_jax)
+    self.assertAllClose(
+      f_tf(x=jnp.zeros(3)),  # Call with kwargs.
```

Comment:
This test fails with JAX_ENABLE_X64=True. I think that you may need to force `jnp.zeros(3, dtype=jnp.float32)`. 

Note that unfortunately the jax2tf test do not run in GitHub, only internally, due to JAX not being ready to take a dependency on tf_nightly, which is needed for jax2tf. This means that test failures only pop up when we try to import the change in Google.